### PR TITLE
Remove clientside commercial performance logging from commercial.js

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

This PR removes complexity from the commercial.js execution steps by pulling out the performance logging logic. The performance data we're recording in the client hasn't been logged anywhere since December 2018 when this PR  https://github.com/guardian/frontend/pull/20745 removed the commercial performance logging experiment from frontend. 

The PR from 2018 removed all the server-side code required to log commercial's client-side performance data (including the `commercial-report` routes used, the experiment configuration, and the admin screens/dashboards). 

In the `commercial.js` we still have all the code in place recording this information but it never gets posted anywhere because [this check](https://github.com/guardian/frontend/compare/gh-remove-commercial-perf-logging?expand=1#diff-642d4b933b687632d3144c9f849047b9L108) now always returns `false`. Since the experiment no longer exists we can now remove all the client-side code used to record these performance metrics.

What this entailed is:
- Removing all the imports from `commercial/modules/dfp/performance-logging` from `commercial.js`
- Removing the now unnecessary `boolean` from the module lists in `commercial.js`. eg. `['cm-prepare-prebid', preparePrebid, true]` becomes `['cm-prepare-prebid', preparePrebid]`.
- Removing the `start()`/`stop()` parameters from the `init` functions

## Screenshots

N/A

## What is the value of this and can you measure success?

Simplifies `commercial.js`, less code to ship, execute and maintain.

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Comment by GHaberis** [comment]
@shtukas the equivalent changes should also be made to `dotcom-rendering-commercial.js` once you begin introducing dependencies.

**Comment by kelvin-chappell** [comment]
Thanks @GHaberis.  I could never work out what could go!

File: static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js [hunk 1]
```diff
@@ -111,10 +111,9 @@ const setPublisherProvidedId = (): void => {
     }
 };
 
-export const init = (start: () => void, stop: () => void): Promise<void> => {
+// onFillAdvertsComplete is used by unit tests to wait for fillAdvertSlots
```
Comment (ripecosta) [inline static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js]: Was this comment left from something you were trying before?

File: static/src/javascripts/projects/commercial/modules/liveblog-adverts.spec.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // @flow
-import { noop } from 'lib/noop';
+// import { noop } from 'lib/noop';
```
Comment (ripecosta) [inline static/src/javascripts/projects/commercial/modules/liveblog-adverts.spec.js]: Would it be better to completely remove this import?

**Review by ripecosta — APPROVED** [review]
Added 2 small comments but otherwise LGTM 👍

File: static/src/javascripts/projects/commercial/modules/liveblog-adverts.spec.js (hunk 2 above)
Comment (GHaberis) [reply static/src/javascripts/projects/commercial/modules/liveblog-adverts.spec.js]: yep, should've cleaned that up!

File: static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js (hunk 1 above)
Comment (GHaberis) [reply static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js]: yep, should've cleaned that up!

**Review by janua — APPROVED** [review]

**Pull request merged**